### PR TITLE
Files for Google Calendar

### PR DESCRIPTION
**Pull request by mcherem**

### CODE REVIEW

File: google-calendar/1.0/antora.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: google-calendar-connector
+version: '1.0.0'
```
Comment (valkyrie69) [inline google-calendar/1.0/antora.yml]:
```suggestion
version: '1.0'
```

File: google-calendar/1.0/antora.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+display_version: 1.0 (Mule 4)
+title: Google Calendar Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables access to Google Calendar.
```
Comment (valkyrie69) [inline google-calendar/1.0/antora.yml]:
```suggestion
    page-component-desc: Enables access to the Google Calendar service.
```

File: google-calendar/1.0/antora.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: com.mulesoft.connectors
+    page-exchange-asset-id: mule-google-calendar-connector
```
Comment (valkyrie69) [inline google-calendar/1.0/antora.yml]: Double check on the Exchange asset. I have noticed with the new connectors, they are using mule4-google-calendar-connector.

File: google-calendar/1.0/modules/ROOT/pages/google-calendar-connector-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,3233 @@
...
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Calendar Id a| String |  Calendar identifier. To retrieve calendar IDS, call the calendarList.list method. If you want to access the primary calendar of the currently logged in user, use the "primary" keyword. |  | x
+| Max Results a| Number |  Maximum number of entries returned on one result page. By default the value is 100 entries. The page size can never be larger than 250 entries. Optional. |  |
+| Pretty Print a| Boolean |  Returns response with indentations and line breaks. |  false |
+| Page Token a| String |  Token specifying which result page to return. Optional. |  |
+| Quota User a| String |  An opaque string that represents a user for quota purposes. Must not exceed 40 characters. |  |
+| Show Deleted a| Boolean |  Whether to include deleted ACLs in the result. Deleted ACLs are represented by role equal to "none". Deleted ACLs will always be included if syncToken is provided. Optional. The default is False. |  false |
```
Comment (valkyrie69) [inline google-calendar/1.0/modules/ROOT/pages/google-calendar-connector-reference.adoc]:
```suggestion
| Show Deleted a| Boolean |  Whether to include deleted ACLs in the result. Deleted ACLs are represented by a role equal to "none". Deleted ACLs are always included if the syncToken is provided. Optional. |  false |
```

File: google-calendar/1.0/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -0,0 +1,50 @@
+= Google Connector 1.0
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Google Calendar (Google Calendar Connector) synchronizes data and automates business processes between Google Calendar and third-party applications, either on-premises or in the cloud.
+
+Google Calendar is a calendar and task planning application that focusses on enhancing personal and team productivity. Using Google calendar, individuals can easily schedule tasks and meetings. They can also share events with colleagues through an easy-to-use calendar interface.
```
Comment (valkyrie69) [inline google-calendar/1.0/modules/ROOT/pages/index.adoc]:
```suggestion
Google Calendar is a calendar and task planning application that focusses on enhancing personal and team productivity. Using Google Calendar, individuals can easily schedule tasks and meetings. They can also share events with colleagues through an easy-to-use calendar interface.
```

**Review by kevintroller — APPROVED** [review]

**Pull request merged**